julia.spdx.json

```diff
@@ -0,0 +1,124 @@
+            "copyrightText": "Copyright (c) 2017-2021: Stefan Karpinski, Kristoffer Carlsson, Fredrik Ekre, David Varela, Ian Butterworth, and contributors: https://github.com/JuliaLang/Pkg.jl/graphs/contributors",
+            "summary": "Julia's package manager, shipped with Julia v1.0 and above"
+        },
+        {
+            "name": "OpenBLAS",
+            "SPDXID": "SPDXRef-OpenBLAS",
+            "downloadLocation": "git+https://github.com/xianyi/OpenBLAS.git",
+            "filesAnalyzed": false,
+            "homepage": "https://www.openblas.net",
+            "sourceInfo": "The git hash of the version in use can be found in the file deps/openblas.version",
+            "licenseConcluded": "BSD-3-Clause",
+            "licenseDeclared": "BSD-3-Clause",
+            "copyrightText": "Copyright (c) 2011-2014, The OpenBLAS Project",
+            "summary": "OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version."
+        },
+        {
+            "name": "libuv",
+            "SPDXID": "SPDXRef-libuv",
+            "supplier": "Organization: julialang.org ()",
+            "originator": "Organization: libuv.org ()",
+            "downloadLocation": "git+https://github.com/JuliaLang/libuv.git",
+            "filesAnalyzed": false,
+            "homepage": "https://libuv.org",
+            "sourceInfo": "The git hash of the version in use can be found in the file deps/libuv.version",
+            "licenseConcluded": "MIT",
+            "licenseDeclared": "MIT",
+            "copyrightText": "Copyright (c) 2015-present libuv project contributors",
+            "summary": "libuv is a multi-platform support library with a focus on asynchronous I/O. It was primarily developed for use by Node.js, but it's also used by Luvit, Julia, pyuv, and others.",
+            "comment": "The Julia project has forked libuv and maintains their own repository of the code"
+        },
+        {
+            "name": "zlib",
+            "SPDXID": "SPDXRef-zlib",
+            "downloadLocation": "git+https://github.com/madler/zlib.git",
+            "filesAnalyzed": false,
+            "homepage": "https://zlib.net",
+            "sourceInfo": "The git hash of the version in use can be found in the file deps/zlib.version",
+            "licenseConcluded": "Zlib",
+            "licenseDeclared": "Zlib",
+            "copyrightText": "Copyright (C) 1995-2017 Jean-loup Gailly and Mark Adler",
+            "summary": "A massively spiffy yet delicately unobtrusive compression library."
+        },
+        {
+            "name": "patchelf",
+            "SPDXID": "SPDXRef-patchelf",
+            "downloadLocation": "git+https://github.com/NixOS/patchelf.git",
+            "filesAnalyzed": false,
+            "homepage": "https://nixos.org/patchelf.html",
+            "sourceInfo": "The version in use can be found in the file deps/Versions.make",
+            "licenseConcluded": "GPL-3.0-only",
+            "licenseDeclared": "GPL-3.0-only",
```

Comment:
From reading the [license in the README](https://github.com/NixOS/patchelf#license), it looks like the `licenseConcluded` and `licenseDeclared` should be `GPL-3.0-or-later`.

Reply:
Thanks!  I had missed that bit in the README and only looked at the COPYING file where that wasn't clear to me.